pom.xml

```diff
@@ -17,6 +17,8 @@
     <packaging>pom</packaging>
 
     <properties>
+        <!--suppress XmlUnresolvedReference --> <!-- git.commit.time is generated by the git-commit-id-maven-plugin-->
```

Comment:
What is the `suppress XmlUnresolvedReference` for? I don't see any XML entity refs...?

I also feel that a comment that we're setting the output timestamp for reproducible build purposes would help

Reply:
It made IntelliJ stop complaining that git.commit.id is unresolved